rocketpool/api/node/send.go

```diff
@@ -79,6 +79,10 @@ func canNodeSend(c *cli.Context, amountWei *big.Int, token string, to common.Add
 	if strings.HasPrefix(token, "0x") {
 		tokenAddress := common.HexToAddress(token)
 
+		if to == tokenAddress {
```

Comment:
`strings.EqualFold()` here, so you don't have case sensitivity.

Reply:
Just noticed these are common.Address, which can be compared with `bytes.Equal()`. Given that it's [40]byte under the hood, `==` technically should also work, but I don't like it because if we change them to pointers it breaks